tasks/src/test/java/com/redhat/parodos/tasks/TibcoWorkFlowTaskTest.java

```diff
@@ -0,0 +1,73 @@
+// @formatter:off
+package com.redhat.parodos.tasks;
+
+import com.redhat.parodos.workflow.context.WorkContextDelegate;
+import com.redhat.parodos.workflow.exception.MissingParameterException;
+import com.redhat.parodos.workflows.work.WorkContext;
+import com.redhat.parodos.workflows.work.WorkReport;
+import com.redhat.parodos.workflows.work.WorkStatus;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import javax.jms.JMSException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+public class TibcoWorkFlowTaskTest {
+
+	String caFile = "/cafile";
+	String url = "ssl://localhost:7222";
+	String username = "username";
+	String passowrd = "password";
+	String topic = "topic";
+	String message = "test message";
+	Tibjms tibjms = Mockito.mock(Tibjms.class);
+	TibcoWorkFlowTask task = new TibcoWorkFlowTask(tibjms, url, caFile, username, passowrd);
```

Comment:
It's a good practice to initialize objects in `@Before` method. 
In this way, the object is recreated before each test.

In addition, make these objects private as well.

Reply:
good practice? I don't think so. Not in this case. In the case of simple initialization, we have the Java language that works for us. You don't need to use additional libraries.
Why make them private? No one is going to use this class outside the test. The default is package-private.

Reply:
If you modify the object inside one of tests, then following tests will use a "dirty" object.
Regarding the `private` = It's a good practice to start to use the narrow privilege and the increase it if needed (so, `private` is more restrictive then `package-private`). Ideally, a good practice is to create immutable classes, but I understand that we can be flexible in this.

Reply:
The Before does not solve it. It only happens once in the beginning. If test A changes something then test B will see a dirty obj.

Reply:
fixed